test/context/yaml/prompts.test.ts

```diff
@@ -18,29 +18,50 @@ describe('#YAML context prompts', () => {
         customText:
           en:
             login:
-              description: text in english
-              title: this is title
-              buttonText: Button text
+              login:
```

Comment:
Not too sure about this `login: { login` thingie. Seems we're doing this mostly for all the different mfa types. What would prevent us from reducing one level and just stop grouping the prompt types?

Reply:
Took this offline and we agreed to leave it as is as it's closer to how the API works as well.